Week-3/1_Calculator/Calculator.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+contract Calculator {
+
+    function addition (uint256 num1, uint256 num2) external pure returns(uint256) {
+        return num1 + num2;
+    }
+
+    function subtraction (int256 num1, int256 num2) external pure returns(int256) {
+        return num1 - num2;
+    }
+
+    function multiplication (uint256 num1, uint256 num2) external pure returns(uint256) {
+        return num1 * num2;
+    }
+
+    function division (uint256 num1, uint256 num2) external pure returns(uint256) {
+        require(num2 != 0, "Can't divide by 0");
+        return num1 / num2;
+    }
+
+    function modulo (uint256 num1, uint256 num2) external pure returns(uint256) {
+        require(num2 != 0, "Can't divide by 0");
+        return num1 % num2;
+    }
+}
```

Comment:
Always add a new line at the end of files

Reply:
Done